### PR TITLE
[keyboard] Adding OSAv2 numpad

### PR DESCRIPTION
**Pull request by blindassassin111**
## Description

Adding support for the MX OSAv2 numpad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/viktus/osav2_numpad/info.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": true
+    },
+    "bootmagic": {
+        "matrix": [0,0]
+    },
```
Comment (waffle87) [inline keyboards/viktus/osav2_numpad/info.json:24]:
```suggestion
```

File: keyboards/viktus/osav2_numpad/info.json [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
```
Comment (waffle87) [inline keyboards/viktus/osav2_numpad/info.json:19]: Is there a reason for these two to be disabled?

Comment (blindassassin111) [reply keyboards/viktus/osav2_numpad/info.json:19]: Was from when I was removing everything from rules.mk. I had this in two other info.JSON's for boards that merged recently and it was never brought up, so didn't notice.

Comment (drashna) [reply keyboards/viktus/osav2_numpad/info.json]: Generally, we like having extra keys and mousekeys enabled, as they're commonly used. And the QMK Configurator doesn't have a way to toggle the features, so it uses whatever these are defaulted to. (eg, it's easier to have enabled)

**Review by waffle87 — APPROVED** [review]

**Comment by blindassassin111** [comment]
Any changes still needed?

File: keyboards/viktus/osav2_numpad/info.json [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        "animations": {
+            "breathing": true,
+            "rainbow_swirl": true,
+            "twinkle": true
+        }
+    },
+    "layouts": {
+        "LAYOUT_all_split": {
+            "layout": [
+                { "label": "K00", "matrix": [0, 0], "w": 1, "x": 0, "y": 0 },
```
Comment (fauxpark) [inline keyboards/viktus/osav2_numpad/info.json:43]:
```suggestion
                { "label": "K00", "matrix": [0, 0], "x": 0, "y": 0 },
```
etc

Comment (blindassassin111) [reply keyboards/viktus/osav2_numpad/info.json:43]: Would this apply to all boards with `"w": 1`? Unsure as it was fine in all of the other PRs I have done in the past month.

Comment (fauxpark) [reply keyboards/viktus/osav2_numpad/info.json:43]: The default width and height of a key is 1u, so they can be omitted unless they are != 1.

**Review by tzarc — APPROVED** [review]

**Pull request merged**